src/sxastarter/.env

```diff
@@ -47,3 +47,8 @@ FETCH_WITH=GraphQL
 #DEBUG=sitecore-jss:layout
 # Or everything BUT layout service logs:
 #DEBUG=sitecore-jss:*,-sitecore-jss:layout
+
+#Set variables for PERSONALIZATION
+NEXT_PUBLIC_CDP_TARGET_URL=http://localhost:9000
+NEXT_PUBLIC_CDP_CLIENT_KEY=test
+NEXT_PUBLIC_CDP_POINTOFSALE=test
```

Comment:
Missing some values here (timeouts), and please leave in the comments. We may have some follow-up adjustments to this, but should be able to use this for now. 
```suggestion
# Your Sitecore CDP API target
# Replace with your API target specific to your data center region
NEXT_PUBLIC_CDP_TARGET_URL=https://api-engage-dev.sitecorecloud.io

# Your Sitecore CDP client key
NEXT_PUBLIC_CDP_CLIENT_KEY=

# Your Sitecore CDP point(s) of sale
# Can be provided as a single value (mypoint.com) or a multi-value JSON with locales ({"en":"en.mypoint.com","fr":"fr.mypoint.com"} etc)
NEXT_PUBLIC_CDP_POINTOFSALE=

# Timeout for CDP requests to respond within (ms). Default is 250.
PERSONALIZE_MIDDLEWARE_CDP_TIMEOUT=

# Timeout for edge endpoint (ms). Default is 250.
PERSONALIZE_MIDDLEWARE_EDGE_TIMEOUT=
```